### PR TITLE
fix: add link fancy ui

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR I add a UI for displaying links in a more fancy manner:

Screenshot 2024-05-08 at 11 32 55

### CODE REVIEW

File: src/Card/constants.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const CARD_HEIGHT = '76px';
```
Comment (pyphilia) [inline src/Card/constants.ts:1]: I noticed this wasn't a constant to the theme spacing, I forgot why it was an issue. I personally changed it to 80.

**Review by pyphilia — APPROVED** [review]
LGTM 🦀

File: src/Card/constants.ts (hunk 1 above)
Comment (spaenleh) [reply src/Card/constants.ts:1]: I took this from the designs directly. We can change it, but maybe it should be the same for all "cards" ?

File: src/Card/components/CardThumbnail.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}: CardThumbnailProps): JSX.Element => {
+  const theme = useTheme();
+  if (thumbnail) {
+    return (
+      <Thumbnail
+        maxWidth={CARD_HEIGHT}
+        maxHeight={CARD_HEIGHT}
+        url={thumbnail}
+        alt={alt}
+        // maxHeight='100%'
```
Comment (ReidyT) [inline src/Card/components/CardThumbnail.tsx:28]: To remove ?

**Review by ReidyT — APPROVED** [review]
Nice design ! LGTM 😆

**Review by mariembencheikh — APPROVED** [review]

File: src/Card/constants.ts (hunk 1 above)
Comment (spaenleh) [reply src/Card/constants.ts:1]: @pyphilia what should be settle on ?

File: src/items/withCaption.tsx [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
+  switch (alignment) {
+    case Alignment.Right:
+      return 'flex-end';
+    case Alignment.Center:
+      return 'center';
+    case Alignment.Left:
+    default:
+      return 'flex-start';
+  }
+};
```
Comment (spaenleh) [inline src/items/withCaption.tsx:32]: This is experimental and most likely is going to be moved to sdk when implementing the feature in the builder.

File: src/theme.ts [hunk 4]
```diff
@@ -5,7 +5,13 @@ import { Context } from '@graasp/sdk';
 
 export const PRIMARY_COLOR = '#5050d2';
 export const SECONDARY_COLOR = '#FFFFFF';
+export const DEFAULT_TEXT_PRIMARY_COLOR = '#313131';
+export const DEFAULT_TEXT_SECONDARY_COLOR = '#71717A';
+export const DEFAULT_ACTIVE_ACTION_COLOR = '#777578';
+export const DEFAULT_LIGHT_PRIMARY_COLOR = '#E4DFFF';
```
Comment (spaenleh) [inline src/theme.ts:11]: Here I explicitly set primary and secondary text colors that do not use transparency, as this is the most effective way of making the lucid icons look good. As they have overlaping paths, colors with opacities make these show.

File: src/items/withCaption.tsx (hunk 3 above)
Comment (pyphilia) [reply src/items/withCaption.tsx:32]: add in comments.

File: src/theme.ts (hunk 4 above)
Comment (pyphilia) [reply src/theme.ts:11]: add in comments.

**Review by pyphilia — APPROVED** [review]

File: src/items/LinkItem.tsx [hunk 5]
```diff
@@ -29,18 +29,13 @@ export type LinkItemProps = {
...
-   */
-  saveButtonId?: string;
   /**
-   * id of the cancel button
+   * Thumbnail url of the item
    */
-  cancelButtonId?: string;
+  thumbnail?: string;
+  loadingMessage?: string;
+  // openLinkMessage?: string;
```
Comment (ReidyT) [inline src/items/LinkItem.tsx:37]: remove ?

**Pull request merged**